### ISSUE TITLE
privateTokens.map is not a function   

### ISSUE THREAD

**Issue by webdevoxo** [harlan-zw/request-indexing#17]
On doing `pnpm i` or `pnpm dev`:

```Lockfile is up to date, resolution step is skipped
Already up to date

> requestindexing.com@0.0.1 postinstall /var/www/request-indexing
> nuxt prepare

ℹ Using default Tailwind CSS file                                                                                                                                                       nuxt:tailwindcss 9:36:34 AM

 ERROR  privateTokens.map is not a function                                                                                                                                                               9:36:36 AM

  at exports.default.defineNuxtConfig.modules (nuxt.config.ts:35:66)
  at installModule (node_modules/.pnpm/@nuxt+kit@3.10.1/node_modules/@nuxt/kit/dist/index.mjs:2415:101)
  at async initNuxt (node_modules/.pnpm/nuxt@3.10.1_@vercel+kv@1.0.1_eslint@8.56.0_typescript@5.3.3_vite@5.1.1/node_modules/nuxt/dist/index.mjs:4018:7)
  at async loadNuxt (node_modules/.pnpm/nuxt@3.10.1_@vercel+kv@1.0.1_eslint@8.56.0_typescript@5.3.3_vite@5.1.1/node_modules/nuxt/dist/index.mjs:4116:5)
  at async loadNuxt (node_modules/.pnpm/@nuxt+kit@3.10.1/node_modules/@nuxt/kit/dist/index.mjs:2570:19)
  at async Object.run (node_modules/.pnpm/nuxi@3.10.0/node_modules/nuxi/dist/chunks/prepare.mjs:56:18)
  at async runCommand$1 (node_modules/.pnpm/nuxi@3.10.0/node_modules/nuxi/dist/shared/nuxi.4fde776c.mjs:1648:16)
  at async runCommand$1 (node_modules/.pnpm/nuxi@3.10.0/node_modules/nuxi/dist/shared/nuxi.4fde776c.mjs:1639:11)
  at async runMain$1 (node_modules/.pnpm/nuxi@3.10.0/node_modules/nuxi/dist/shared/nuxi.4fde776c.mjs:1773:7) ```


Where do I get these 2 variables from pls ?

env.NUXT_OAUTH_POOL
env.NUXT_OAUTH_PRIVATE_POOL

**Comment by AntoineKM** [comment]
I'm going to sort it out

**Comment by AntoineKM** [comment]
I had the same problem, it should be solved in #19
